.eslint.js

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const colors = require('colors');
+const eslint = require('eslint').CLIEngine;
+
+var filesToLint = [
+    'chrome/background.js',
+    'firefox/background.js',
+    'firefox-mobile/background.js',
+    'opera/background.js',
+    'test.js'
+];
+
+var lintConfig = new eslint({
+    fix: true,
+    useEslintrc: false,
+    env: ["browser", "es6"],
+    extends: "eslint:recommended",
+    globals: [
+        "chrome: true",
+        "content: true",
+        "Components: true",
+        "XPCOMUtils: true",
+        "CustomizableUI: true",
+        "Services: true",
+        "addMessageListener: true",
+        "removeMessageListener: true",
+        "sendAsyncMessage: true"
```

Comment:
In https://github.com/webcompat/webcompat-reporter-extensions/pull/63, we're going to remove the "firefox-bootstrap" directory, so we won't need any of these globals I think. I'll merge that, and maybe you can double check after rebasing?